docker/README.md

```diff
@@ -0,0 +1,17 @@
+Overview of how the ray images are built
+	 .─────────────────────────────────────────────────────.
+	(      (:-gpu)              ray-ml         (-cpu)       )
+	 `─────────────────────────────────────────────────────'
+	 .─────────────────────────────────────────────────────.
+	(      (:-gpu)              ray            (-cpu)       )
+	 `─────────────────────────────────────────────────────'
+
+	 .─────────────────────────────────────────────────────.
+	(      (:-gpu)            ray-deps         (-cpu)       )
+	 `─────────────────────────────────────────────────────'
+	 .─────────────────────────────────────────────────────.
+	(      (:-gpu)           base-deps         (-cpu)       )
+	 `─────────────────────────────────────────────────────'
+	 .───────────────────────.     .───────────────────────.
+	(       nvidia/cuda       )   (      ubuntu/focal       )
+	 `───────────────────────'     `───────────────────────'
```

Comment:
output formatting looks a little funky, maybe put this inside of code tags?
``````suggestion
Overview of how the ray images are built

```
         .─────────────────────────────────────────────────────.
	(      (:-gpu)              ray-ml         (-cpu)       )
	 `─────────────────────────────────────────────────────'
	 .─────────────────────────────────────────────────────.
	(      (:-gpu)              ray            (-cpu)       )
	 `─────────────────────────────────────────────────────'

	 .─────────────────────────────────────────────────────.
	(      (:-gpu)            ray-deps         (-cpu)       )
	 `─────────────────────────────────────────────────────'
	 .─────────────────────────────────────────────────────.
	(      (:-gpu)           base-deps         (-cpu)       )
	 `─────────────────────────────────────────────────────'
	 .───────────────────────.     .───────────────────────.
	(       nvidia/cuda       )   (      ubuntu/focal       )
	 `───────────────────────'     `───────────────────────'
```
``````

Reply:
Let me know if my adding an extra line didn't fix it ?

Reply:
that fixed it!